spec/unit/installer/xcode/pods_project_generator_spec.rb

```diff
@@ -264,8 +264,11 @@ def test_extension_target(symbol_type)
               mock_native_target = mock('CoconutLib')
               mock_test_native_target = mock('CoconutLib-Unit-Tests')
 
-              dependent_target = mock('dependent-target', :should_build? => true, :native_target => 'DependentNativeTarget')
-              test_dependent_target = mock('dependent-test-target', :should_build? => true, :native_target => 'TestDependentNativeTarget')
+              dependent_native_target = mock('DependentNativeTarget')
```

Comment:
no real changes here, just updated test to actually use a mock object as a representation of the native target instead of just a string.